src/pyramid/security.py

```diff
@@ -280,19 +287,68 @@ class ACLAllowed(ACLPermitsResult, Allowed):
     """
 
 
+class SecurityAPIMixin(object):
+    @property
+    def identity(self):
```

Comment:
What about `authorized_user`?

Reply:
My understanding was we were just doing `identity`.

Reply:
Well it's an opaque object, yeah, but on the request I would prefer that it to be something more security-oriented than just `request.identity` because that feels fairly meaningless. For example, we have other systems like the debugtoolbar or tracing libraries that assign a `request.id` to a request. It's just an overloaded term and some form of namespacing would help. So `authorized_user`, `authorized_userid`, `authorized_identity`, `security_identity` or something would be better IMO. Definitely open to bike-shedding some ideas but I don't think `request.identity` is clear enough.

Reply:
I don't find `identity` to be a problem.  If we are going for something more explicit, I'd vote for `authenticated_user` (too close to deprecated `authenticated_userid`?) or `authenticated_identity` or `secure_identity`.  I personally don't like `authorized_*` because it implies it's somehow involved with the authorization mechanisms, but it's strictly in the realm of identity/authentication.